packages/block-editor/src/components/global-styles/shadow-panel-components.js

```diff
@@ -89,8 +88,7 @@ export function ShadowIndicator( { type, label, isActive, onSelect, shadow } ) {
 			} ) }
 			render={
 				<Button
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
| Before | After     |
| :-------- | :------- |
| ![shadow-indicator-before](https://github.com/user-attachments/assets/605da444-c9e3-4b2f-9dd3-f40a1d6b0ce3) | ![shadow-indicator-after](https://github.com/user-attachments/assets/d24c5fef-7bf8-4a15-8eb7-51625259899a)|

It looks a bit stretched to me after setting it to `40px`. I tried using the `small` size, but then it appeared a bit too shrunken. Let me know if any adjustments are needed.

Reply:
@WordPress/gutenberg-design ?

Seems to me like we just want normal `<button>`s here, not sure why use `Button` if these are fully custom.

Reply:
It seems that the Shadow Palette has used the `Button` component since it was first implemented: https://github.com/WordPress/gutenberg/pull/46502/files#diff-d3db145a5a2d621586d0b1740cc8b403e58afac79bfe418bf7752fa3af88dc2bR164-R171

I don't know why the `Button` component was used, but based on [this comment](https://github.com/WordPress/gutenberg/blob/e9d31c255ce0dbf20d5ed0b6484ec74813393521/packages/block-editor/src/components/global-styles/style.scss#L38), it seems that we were trying to imitate the style of the color palette (`CircularOptionPicker.Option` component).

In fact, the `CircularOptionPicker.Option` component is also implemented internally as a `Button` component and is fully customized ([source](https://github.com/WordPress/gutenberg/blob/e9d31c255ce0dbf20d5ed0b6484ec74813393521/packages/components/src/circular-option-picker/style.scss#L33-L40))

Reply:
For this PR it's probably best to keep those buttons _visually_ looking the same as before, regardless of what component is used. A `button` element rather than the _component_ may be a fine solution there. 

Longer term, we should find a better design for these that's clearer.

Reply:
It would be good to avoid the arbitrary `26px` sizing and use of the more standardised options (`24`, `32`, or `40`).

Reply:
> For this PR it's probably best to keep those buttons visually looking the same as before, regardless of what component is used. A button element rather than the component may be a fine solution there.

Agreed. For heavily customized components with lots of style overrides, it may no longer make sense to use the underlying component. A `button` would be a better way to do it as @DaniGuardiola initially suggested. And if we end up having multiple similar usages, we can always componentize it.

Reply:
Hi @tyxla I have converted it to normal HTML buttons.

Reply:
To maintain the previous appearance and functionality, I think the following problems need to be solved:

- Apply `none` to the button's background color; otherwise, the browser's default gray background color (in Chrome) will be applied.
- The checkmark position is incorrect when the shadow is selected and focused. We need to apply `display:flex` or `display:inline-flex` and `align-items:center` to the button.
- We should use the `Tooltip` component to display text instead of the `title` attribute.
- Two attributes, `label` and `aria-label`, are applied, but only `aria-label` should be needed.
- The `__next40pxDefaultSize` prop will no longer be necessary.
- We might need `apperance: none` to reset the browser styles just to be sure.

| trunk | This PR |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/170aab66-9dab-47f1-87b5-d44b718b4f25) | ![image](https://github.com/user-attachments/assets/7965fdcb-e298-4cf1-a48c-ddcaf304b51f)|

Reply:
I also noticed that wrapping the `button` with a `Tooltip` breaks the `Composite`'s keyboard navigation, so we need to wrap the entire `Composite.Item` with the `Tooltip`.

☝️ @ciampo Did you know about this quirk? Not immediately sure why that is. The `Tooltip` embedded in a `Button` component doesn't break Composite.

Reply:
Thanks for the update. I think it perfectly maintains its functionality and style 👍

Reply:
Documented the Composite + Tooltip issue at #65615.